### PR TITLE
Student home

### PR DESCRIPTION
**Pull request by Ishanliv99**
- Complete basic Student Home Page with Modals
- Added feature to add modals

Run `yarn` after pulling

### CODE REVIEW

File: App.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import App from './App';
```
Comment (abiralacharya) [inline App.test.js]: separate it with a line

**Review by abiralacharya — COMMENTED** [review]
use branch name in all lower case

**Review by yushghimire — COMMENTED** [review]
1. Cannot go back after in add class modal
2. Classes are not center aligned 
3. Empty submit is allowed in classes added

**Review by abiralacharya — APPROVED** [review]

**Review by abiralacharya — DISMISSED** [review]
Nice!!

File: src/Index.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import ClassList from './ClassList';
+import AddButton from './AddButton';
+import SemesterPicker from './SemesterPicker';
+
+export default class Index extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      allClasses: ['CS 101', 'ECON 303', 'HIST 201'],
```
Comment (yushghimire) [inline src/Index.js]: implement ladder in state

**Review by yushghimire — CHANGES_REQUESTED** [review]
implement ladder

**Review by yushghimire — APPROVED** [review]

**Review by abiralacharya — APPROVED** [review]

**Pull request merged**